src/05-derivatives/03-imaging.md

```diff
@@ -0,0 +1,368 @@
+REQUIRED for GIFTI files. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_hemi-L_dseg.label.gii
+            sub-001_hemi-R_dseg.label.gii
+```
+
+The REQUIRED extension for CIFTI parcellations is `.dlabel.nii`. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_dseg.dlabel.nii
+            sub-001_dseg.dlabel.nii
+```
+
+### Anatomical Labels
+
+BIDS supplies a standard, generic label-index mapping, defined in the table
+below, that contains common tissue classes and can be used to map segmentations
+(and parcellations) between lookup tables.
+
+| Integer value | Description             | Abbreviation (label) |
+| ------------- | ----------------------- | -------------------- |
+| 0             | Background              | BG                   |
+| 1             | Gray Matter             | GM                   |
+| 2             | White Matter            | WM                   |
+| 3             | Cerebrospinal Fluid     | CSF                  |
+| 4             | Gray and White Matter   | GWM                  |
+| 5             | Bone                    | B                    |
+| 6             | Soft Tissue             | ST                   |
+| 7             | Non-brain               | NB                   |
+| 8             | Lesion                  | L                    |
+| 9             | Cortical Gray Matter    | CGM                  |
+| 10            | Subcortical Gray Matter | SCGM                 |
+| 11            | Brainstem               | BS                   |
+| 12            | Cerebellum              | CBM                  |
+
+These definitions can be overridden (or added to) by providing custom labels in
+a sidecar `<matches>.tsv` file, in which `<matches>` corresponds to segmentation
+filename.
+
+Example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_dseg.nii.gz
+            sub-001_space-orig_dseg.tsv
+```
+
+Definitions can also be specified with a top-level `dseg.tsv`, which propagates to
+segmentations in relative subdirectories.
+
+Example:
+
+```Text
+pipeline/
+    dseg.tsv
+    sub-001/
+        anat/
+            sub-001_space-orig_dseg.nii.gz
+```
+
+These TSV lookup tables contain the following columns:
+
+| Column name  | Description                                                             |
+| ------------ | ----------------------------------------------------------------------- |
+| index        | REQUIRED. The label integer index                                       |
+| name         | REQUIRED. The unique label name                                         |
+| abbreviation | OPTIONAL. The unique label abbreviation                                 |
+| mapping      | OPTIONAL. Corresponding integer label in the standard BIDS label lookup |
+| color        | OPTIONAL. Hexadecimal. Label color for visualization                    |
```

Comment:
```suggestion
| color        | OPTIONAL. Hexadecimal. Label color for visualization                    |
| mapping      | OPTIONAL. Corresponding integer label in the standard BIDS label lookup |
```
stylistic change to match column order in the example below.

Reply:
the more specific comment is to perhaps expand on how non-existent values should be encoded since the BIDS table only has 12 segments. most of these are likely to have more.